src/translations/en.ts

```diff
@@ -1474,6 +1474,7 @@ export default {
 		"yourCalendars_label": "Your calendars",
 		"yourFolders_action": "YOUR FOLDERS",
 		"yourMessage_label": "Your message",
-		"you_label": "You"
+		"you_label": "You",
+		"deviceEncryptionSaveCredentialsHelpText_msg": "Device encryption will protect your credentials when you save them to your device. You can do this the next time you log in.",
```

Comment:
this seems slightly unclear. maybe something like 
* "Select an unlock method to protect Tutanota credentials that were saved at Login."
* "Wähle eine Entsperrmethode, um beim Login gespeicherte Tutanota-Zugangsdaten zu schützen."
* "Wählen Sie eine Entsperrmethode, um beim Login gespeicherte Tutanota-Zugangsdaten zu schützen."
 would be better?

The fact that the credentials are encrypted seems already implied by the label of the setting.

Reply:
actually Hanna has made some texts of her own in Phrase, that was just my placeholder. It'll be overwritten when we pull the translations.

Reply:
okay, we still need de and de_sie for the tests to run I think. just pasting the english version should be enough